apps/gelato-vrf/README.md

```diff
@@ -0,0 +1,38 @@
+# How To #10 on Gelato: VRF Jump Start
+
+This repository provides example contracts and instructions for setting up Verifiable Random Function (VRF) on Gelato, utilizing Gelato VRF.
+
+[![Video Walkthrough](https://img.youtube.com/vi/cUPjQYoH2OE/0.jpg)](https://youtu.be/cUPjQYoH2OE)
+
+
+## Overview
+
+Verifiable Random Function (VRF) is a cryptographic primitive that generates unpredictable random numbers. Integrating VRF into Gelato allows for various decentralized applications (DApps) to leverage secure, on-chain randomness.
+
+This repository includes contracts and configurations necessary for implementing VRF functionality on Gelato, making it easy for developers to integrate random number generation into their projects.
+
+## Contracts
+
+There are two types of VRF contracts available in this repository:
+
+1. Gelato VRF
+
+> [!NOTE]  
+> Check the deployed addresses for each network in the deployments folder.
```

Comment:
Where is the deployments folder? If this is created after performing a certain step, then this should specify.

Reply:
I've clarified in the README that the deployments folder is automatically generated at the root of the gelato-vrf/ directory after each deployment.